DNN Platform/Library/Services/Upgrade/DnnInstallLogger.cs

```diff
@@ -67,11 +63,11 @@ public static void InstallLogInfo(object message)
 
 		private static void LogInstaller(string logType, string message)
 		{
-			string logFile = "InstallerLog" + DateTime.Now.Year.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Month.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Day.ToString(CultureInfo.InvariantCulture) + ".resources";
-			string logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
+		    var logFile = InstallController.Instance.InstallerLogName;
+			var logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
 			using (var writer = new StreamWriter(logfilePath, true))
 			{
-				writer.WriteLine(DateTime.UtcNow.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
+				writer.WriteLine(DateTime.Now.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
```

Comment:
I think all dates should be in the UTC time, and translated for display.  If QA has a problem with the way dates are stored then they need to create a proper Jira so that it can be discussed by the PMs. Please back out that single change so we can merge the rest of this fix.  I am happy to deal with the date issue on a separate Jira item.